### PR TITLE
Clean up language split. Close minor issues.

### PR DESCRIPTION
**Pull request by austin-hilberg**

### CODE REVIEW

File: contents/queries/05-Operators.rst [hunk 1]
```diff
@@ -83,7 +83,7 @@ Join compatibility
 -------------------
 All binary operators with other tables as their two operands require that the operands be **join-compatible**, which means that:
 
-1. All common attributes in both operands (attributes with the same name) must be part of the primary key or of a foreign key.
+1. All common attributes in both operands (attributes with the same name) must be part of either the primary key or a foreign key.
```
Comment (dimitri-yatsenko) [inline contents/queries/05-Operators.rst]: Although this rule correctly reflects the current behavior of datajoint-python, this rule will be replaces with the rule specified in the theory preprint, which relies on the concept of homologous namesake attributes for matching.

Comment (dimitri-yatsenko) [reply contents/queries/05-Operators.rst]: Perhaps this should be mentioned in a parenthetical remark.

Comment (dimitri-yatsenko) [inline contents/queries/05-Operators.rst:86]: actually, the currently implemented rule is that all the namesake attributes must be be part of the primary key in either of the operands.  We should submit an issue to datajoint-python and datajoint-matlab to change the behavior to how it's documented here.

**Review by dimitri-yatsenko — APPROVED** [review]

File: contents/queries/05-Operators.rst (hunk 1 above)
Comment (austin-hilberg) [reply contents/queries/05-Operators.rst:86]: I have created corresponding issues.

**Pull request merged**